_data/clients/atalk.yml

```diff
@@ -1,6 +1,8 @@
 name: aTalk
 url: https://play.google.com/store/apps/details?id=org.atalk.android&hl=en
+tracking_issue: https://github.com/cmeng-git/atalk-android/issues
```

Comment:
The `tracking_issue` link is meant to point to a specific discussion about the implementation of OMEMO in a client. There doesn't seem to be an issue already, so it should probably be removed from the YAML. (It isn't worth it to make an issue just to link to it.)

```suggestion
```

Reply:
Sorry, I thought the tracking_issue is a link to which user may input newly encountered problems if found. It is quite tedious for some one to generate a pull request if new omemo issue is found or the found omemo problem has been resolved.
This permanent tracking_issue link allows user to check the aTalk current development status.

If the link really serves no purpose in the intent of omemo-top site, would you be able to just remove it.

Reply:
Hi @cmeng-git! Thanks for your contribution! As @evaryont suggested – the tracking issue is for tracking the actual OMEMO implementation. If the OMEMO status is "done", the tracking issue isn't even linked, so you could just very well leave it out. :)